src/main/java/com/softserve/actent/model/entity/User.java

```diff
@@ -61,17 +54,18 @@
     private String password;
 
     @NonNull
-    @NotNull(message = StringConstants.EMPTY_USER_BIRTH_DATE)
-    @Column(nullable = false)
+    @JsonManagedReference
+    @Column
     private LocalDate birthDate;
 
     @NonNull
     @ManyToOne
+    @JsonManagedReference
     private Image avatar;
 
     @NonNull
-    @NotNull(message = StringConstants.EMPTY_USER_LOCATION)
     @ManyToOne
+    @JsonManagedReference
```

Comment:
Try to properly use mapper instead of jackson dependencies.

Reply:
done